addons/common/stringtable.xml

```diff
@@ -160,141 +172,165 @@
             <German>Fluggerät</German>
             <Polish>Pojazd latający</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Filter">
             <English>Filter</English>
+            <French>Filtre</French>
             <Russian>Фильтр</Russian>
             <German>Filter</German>
             <Polish>Filtr</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Units">
             <English>Units</English>
+            <French>Unités</French>
             <Russian>Солдаты</Russian>
             <German>Einheiten</German>
             <Polish>Jednostki</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Vehicles">
             <English>Vehicles</English>
+            <French>Véhicules</French>
             <Russian>Машины</Russian>
             <German>Fahrzeuge</German>
             <Polish>Pojazdy</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Static">
             <English>Static</English>
+            <French>Statique</French>
             <Russian>Турели</Russian>
             <German>Statische Waffen</German>
             <Polish>Obiekty statyczne</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Nearest">
             <English>Nearest</English>
+            <French>Le plus proche</French>
             <Russian>Ближайший</Russian>
             <German>Am nächsten</German>
             <Polish>Najbliższe</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Speed">
             <English>Speed</English>
+            <French>Vitesse</French>
             <Russian>Скорость</Russian>
             <German>Geschwindigkeit</German>
             <Polish>Prędkość</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Range">
             <English>Range</English>
+            <French>Portée</French>
             <Russian>Дистанция</Russian>
             <German>Reichweite</German>
             <Polish>Zasięg</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Radius">
             <English>Radius</English>
+            <French>Rayon</French>
             <Russian>Радиус</Russian>
             <German>Radius</German>
             <Polish>Promień</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Height">
             <English>Height</English>
+            <French>Hauteur</French>
             <Russian>Высота</Russian>
             <German>Höhe</German>
             <Polish>Wysokość</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Height_Units">
             <English>Height (m)</English>
+            <French>Hauteur (m)</French>
             <Russian>Высота (м)</Russian>
             <German>Höhe (m)</German>
             <Polish>Wysokość (m)</Polish>
         </Key>
         <Key ID="STR_ZEN_Common_Teleport">
             <English>Teleport</English>
+            <French>Téléportation</French>
         </Key>
         <Key ID="STR_ZEN_Common_AutoAddObjects">
             <English>Add Objects to Curators</English>
+            <French>Ajouter les objets aux collections</French>
```

Comment:
Curators means "Zeus" here.

Reply:
So I should use "Zeus", can you explain what "curators" mean?

Reply:
Curator is just a fancy word for Zeus. In German the most common use of curator is for describing a role of someone working at a museum. Maybe it's the same word/meaning in french.

Reply:
> Curator is just a fancy word BI is using for Zeus

I'm not sure if he's an arma player. Looking at profile it looks like it's just a translation for Hacktoberfest (sorry if I'm mistaken). Great job anyway.

Reply:
> > Curator is just a fancy word BI is using for Zeus
> 
> I'm not sure if he's an arma player. Looking at profile it looks like it's just a translation for Hacktoberfest (sorry if I'm mistaken). Great job anyway.

You are right, I love translate a lot of technical documentation. This event is a good reason to find a lot of repositories in needs of translation for me. :) But I'm a player too! Just don't know this mod and I will test it when I can! Ahah

Reply:
> Curator is just a fancy word BI is using for Zeus. In German the most common use of curator is for describing a role of someone working at a museum. Maybe it's the same word/meaning in french.

Thank you for the precision!

Reply:
You're welcome. digitalocean could make a hacktoberfest all year long :)